### ISSUE TITLE
Maintenance of log output

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#93]
The file
`RobotFramework_TestsuitesManagement\Keywords\CSetup.py`
contains the following lines:

```
logger.info(f"Suite Path: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestcasePath}")
logger.info(f"CfgFile Path: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}")
if RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sLocalConfig != '':
    logger.info(f"Local config file: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sLocalConfig}")
logger.info(f"Suite Count: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.iSuiteCount}")
logger.info(f"Total testcases in TestSuite {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sRootSuiteName} \
is: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.iTotalTestcases}")
```

(1)
I would prefer to have all logged paths encapsulated in single quotes (like also done at other positions).

(2)
Instead of "Suite count:" I would prefer "Number of test suites:"

(3)
Instead of "Total testcases in TestSuite" I would prefer "Total number of test cases:"

**Comment by namsonx** [comment]
Hello Holger,

I made the pull-request #99  for this enhancement.

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and already merged with #99
